ShowBuildMenu.bat

```diff
@@ -130,6 +130,7 @@ set LIB_FILES2=
 goto test-setup-generic
 
 :test-setup-generic
+set CFGNAME=
 set /p CFGNAME=Enter a name for your test configuration or press enter to use default name: 
```

Comment:
The first run is able of using default name if user does not input anything. But subsequent runs re-use the last set name, because `CFGNAME` remains defined in the shell till we close and relaunch it.

So I have added a `SET` for ensuring `CFGNAME` is empty before prompting user.